src/modules/Speech_Recognition/Whisper.py

```diff
@@ -21,21 +25,22 @@ def transcribe_with_whisper(audio_path: str, model: str, device="cpu", model_nam
     if model_name is not None:
         print(f"{ULTRASINGER_HEAD} using alignment model {blue_highlighted(model_name)}")
 
-    batch_size = 16  # reduce if low on GPU mem
-    compute_type = (
-        "float16" if device == "cuda" else "int8"
-    )  # change to "int8" if low on GPU mem (may reduce accuracy)
+    if compute_type is None:
+        compute_type = "float16" if device == "cuda" else "int8"
 
     loaded_whisper_model = whisperx.load_model(
-        model, device=device, compute_type=compute_type
+        model, language=language, device=device, compute_type=compute_type
     )
 
     audio = whisperx.load_audio(audio_path)
 
     print(f"{ULTRASINGER_HEAD} Transcribing {audio_path}")
 
-    result = loaded_whisper_model.transcribe(audio, batch_size=batch_size)
-    language = result["language"]
+    result = loaded_whisper_model.transcribe(audio, batch_size=batch_size, language=language)
```

Comment:
I got `TypeError: FasterWhisperPipeline.transcribe() got an unexpected keyword argument 'language'` here. I dont specified a language in cli, so it should be None.

Reply:
This is strange, I can't reproduce this problem.
When I omit the language CLI flag the program runs just fine.

I can't see how this could have happened but maybe try anyways to recreate the venv as some dependencies have changed. The whisperx version is unchanged as far as I can tell.

Reply:
Try. Was my fold.